clang/include/clang/Driver/Action.h

```diff
@@ -793,18 +793,34 @@ class FileTableTformJobAction : public JobAction {
   // <To> from another file table passed as input to this action.
   void addReplaceColumnTform(StringRef From, StringRef To);
 
+  // Replaces a cell in this table with column title <ColumnName> and row <Row>
+  // with the file name passed as input to this action.
+  void addReplaceCellTform(StringRef ColumnName, int Row);
+
   // Renames a column with title <From> in this table with a column with title
   // <To> passed as input to this action.
   void addRenameColumnTform(StringRef From, StringRef To);
 
+  // Specifies that, instead of generating a new table, the transformation
+  // should copy the file in the only remaining row at <ColumnName> into the
+  // output file.
+  void setCopySingleFileColumn(StringRef ColumnName);
+
```

Comment:
Also, please make the 'copy single file column' action a usual Tform. That way it would be much less intrusive change and play more nicely with the overall tool design. This Tform would be required to be the last one in the tform chain the file-table-tform processes.

Reply:
Makes sense to me, especially with the change to the arguments. I have made it a tform and made a check to make sure it is the last transformation. clang-format was a bit aggressive about some of the code around the cases I added for the new tform, but there was no way around it.